### PR TITLE
LPD-37841 Add popover when hovering over the Conflict Icon and match scaled size with Timeline Icon

### PR DESCRIPTION
**Pull request by ctang3**
https://liferay.atlassian.net/browse/LPD-37841

image

### CODE REVIEW

**Comment by ctang3** [comment]
ci:test:relevant

**Comment by ctang3** [comment]
@david-truong Test failures aren't related. Does the attached pic look good to you? Ready to forward if it does.

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js [hunk 1]
```diff
@@ -438,13 +439,16 @@ export default function ChangeTrackingIndicator({
...
-					menuElementAttrs={{style: {maxWidth: '303px'}}}
+				<ClayPopover
+					alightPosition="bottom"
+					onShowChange={setOpenPopover}
+					show={openPopover}
 					trigger={
 						<ClayButton
 							aria-label="conflict-button"
 							className="change-tracking-conflict-button"
+							onMouseOut={() => setOpenPopover(false)}
```
Comment (david-truong) [inline modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js:450]: @ctang3 does this mean the moment you move off the icon the popover disappears?

**Comment by david-truong** [comment]
@ctang3 can we also make the white space around the warning even?

File: modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js (hunk 1 above)
Comment (ctang3) [reply modules/apps/change-tracking/change-tracking-web/src/main/resources/META-INF/resources/publications/js/components/ChangeTrackingIndicator.js:450]: Yeah, there's a very slight delay before the popover disappears after mousing out.

**Comment by ctang3** [comment]
@david-truong Updated to make the whitespace around the warning even. 

image

**Comment by david-truong** [comment]
ci:test:sf

**Comment by david-truong** [comment]
ci:test:publications

**Comment by david-truong** [comment]
ci:test:relevant

**Comment by david-truong** [comment]
ci:forward

**Pull request closed without merging**